css/src/components/form/radio.scss

```diff
@@ -0,0 +1,96 @@
+$radio-circle-size: 1.125rem !default;
```

Comment:
Should the radio circle contain a background color of its own? I believe this would ensure the control would work regardless of where its placed

Reply:
Added `background-color: $body-background;`